modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AspNetCoreServerCodegen.java

```diff
@@ -182,6 +196,21 @@ public AspNetCoreServerCodegen() {
                 "Generates method body.",
                 generateBody);
 
+        addSwitch(OPERATION_IS_ASYNC,
+                "Set methods to async or sync.",
+                operationIsAsync);
```

Comment:
@A-Joshi What about reusing `asyncServer` option in Nancyfx generator?
```
	asyncServer
	    Set to true to enable the generation of async routes/endpoints. (Default: false)
```
Ref: https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/CSharpNancyFXServerCodegen.java#L98

Reply:
I thought about it but it is the operation (method) which is async not necessarily the server. In the future I can see this extended to have a list of operations which are async while leaving others sync etc.

Reply:
👌